Rawl_B502/de_raind_h_érenn.trig

```diff
@@ -28,7 +28,7 @@
 <#ÉoganMár>
     a foaf:Person ;
     irishRel:nomName "Éogan Már" ;
-    owl:sameAs <http://example.com/LL/senchas_síl_ébir.trig#EoganMór>.
+    owl:sameAs <#MugNuadat>.
```

Comment:
You may need to explain this one to me :)

Reply:
Fairly straightforward. Here's the source text:

> Diuissa est Hibernia insola in duas partes compares eter Conn Cétchathach et Éogan Már qui et Mug Nuadat m. Dergthene diximus...